R/lastdose.R

```diff
@@ -133,11 +144,13 @@ NULL
 #'
 #' @export
 lastdose <- function(data, ..., include_ldos = TRUE,
-                     include_tafd = getOption("lastdose.include_tafd", FALSE)) {
+                     include_tafd = getOption("lastdose.include_tafd", FALSE),
+                     include_occ = getOption("lastdose.include_occ", TRUE)) {
   ans <- lastdose_list(data, ...)
```

Comment:
Should `lastdose` pass `include_occ` to `lastdose_list`? It's not strictly necessary due to the `include_occ` condition on line 153, but it'd prevent `lastdose_impl` from doing unnecessary work to generate `occ` just for it to be discarded.

Reply:
Thanks; agree and I think I was aiming for that, but got wires crossed for what was going where.